src/Core/Resolvers/SqlMutationEngine.cs

```diff
@@ -923,6 +957,763 @@ private FindRequestContext ConstructFindRequestContext(RestRequestContext contex
             return dbResultSetRow;
         }
 
+        /// <summary>
+        /// Performs the given GraphQL create mutation operation.
+        /// </summary>
+        /// <param name="entityName">Name of the top level entity</param>
+        /// <param name="parameters">Mutation parameter arguments</param>
+        /// <param name="sqlMetadataProvider">SqlMetadaprovider</param>
+        /// <param name="context">Hotchocolate's context for the graphQL request.</param>
+        /// <param name="multipleInputType">Boolean indicating whether the create operation is for multiple items.</param>
+        /// <returns>Primary keys of the created records (in the top level entity).</returns>
+        /// <exception cref="DataApiBuilderException"></exception>
+        private List<IDictionary<string, object?>> PerformMultipleCreateOperation(
+                string entityName,
+                IMiddlewareContext context,
+                IDictionary<string, object?> parameters,
+                ISqlMetadataProvider sqlMetadataProvider,
+                bool multipleInputType = false)
+        {
+            string fieldName = multipleInputType ? MULTIPLE_INPUT_ARGUEMENT_NAME : SINGLE_INPUT_ARGUEMENT_NAME;
```

Comment:
`fieldName` is too generic. This has more significance than just an ordinary field, right?